scipy/stats/_fit.py

```diff
@@ -164,10 +192,78 @@ def plot(self, ax=None):
             ax.plot(self._data, np.zeros_like(self._data), "*",
                     label='Data', color='C1')
 
-        ax.set_title(f"{self._dist.name} Fit")
+        ax.set_title(f"`{self._dist.name}` Fit Histogram")
         ax.legend(*ax.get_legend_handles_labels())
         return ax
 
+    def _qp_plot(self, ax, fit_params, qq):
+        data = np.sort(self._data)
+        n = len(self._data)
+        a = 0.5  # expose this as a parameter later?
+        # See https://en.wikipedia.org/wiki/Q%E2%80%93Q_plot#Plotting_positions
```

Comment:
How do you mean? We want points that are some measure of central tendency of the order statistics of the reference distribution, or some similar heuristic like this. 

QMC could be used to, say, estimate the means of the order statistics, I suppose.

Reply:
BTW I chose `a = 0.5` from the possibilities because that is what [`fitdistrplus`](https://cran.r-project.org/web/packages/fitdistrplus/vignettes/paper2JSS.pdf) does.

![image](https://user-images.githubusercontent.com/6570539/188803317-839d25ea-b774-4782-b8a7-c9c5a6e0dd67.png)

They do give an option for adjusting `a`, apparently, but I would prefer not to do that in this PR.